althea_kernel_interface/src/exit_server_counter.rs

```diff
@@ -108,7 +107,7 @@ impl KernelInterface {
                 "inet6",
                 "counters",
             ],
-        );
+        ).expect("Failed to create new ipset to swap");
```

Comment:
this can crash if rita crashes during the counter swap, and make it crash every 5 seconds, might be better to ignore the error

Reply:
+1 but I'd also log it with `warn!()`